oss-internship-2020/libtiff/test/defer_strile_writing.cc

```diff
@@ -0,0 +1,327 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <cstdint>
+
+#include "helper.h"
+#include "tiffio.h"  // NOLINT(build/include)
+
+// sapi functions:
```

Comment:
Same as before. In case you want to state that those are the tested functions, please note that down here (I just noticed that this might be your intention).

Reply:
yes, that's what it is for. But I'm not sure if anyone other than me is interested in these notes :)